### PR TITLE
Wrappers: save to pdf via matplotlib 

### PR DESCRIPTION
**Pull request by jenshnielsen**
Add automatic saving of pdf to wrappers.

In addition:

* Change default matplotlib colormap to match pyqtgraph
* Hack around aliasing issue in pcolormesh To see this in action look at 
[031.pdf](https://github.com/qdev-dk/Qcodes/files/841937/031.pdf)
renders correctly in Chrome but has visual gaps in OSX Preview and probably others
as seen in this screenshot:
test

@giulioungaretti

### CODE REVIEW

**Comment by giulioungaretti** [comment]
Karl wants this to save a figure per plot :/

**Comment by giulioungaretti** [comment]
also we want to use the pretty ugly location thing stored in the current experiment wrapper!

**Comment by giulioungaretti** [comment]
and we probably want to make it matplotlib also in the show_num!

**Comment by giulioungaretti** [comment]
Let's figure out if it's possible to save a pdf per sublopt after the do1 and do2d are done, as per comment above! @jenshnielsen

**Comment by jenshnielsen** [comment]
Did not see the last comment have made the change already in the last commit

File: qcodes/plots/pyqtgraph.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pyqtgraph as pg
 import pyqtgraph.multiprocess as pgmp
+from qtpy import QtWidgets
```
Comment (giulioungaretti) [inline qcodes/plots/pyqtgraph.py:7]: @jenshnielsen  actually this broke  everything ! We need to use the on from pyqtgraph

**Comment by giulioungaretti** [comment]
Note to non tired self: from pyqtgraph.Qt import QtWidgets
And the .reg file hack does not actually work :/

**Pull request merged**